src/test-jdk14/java/com/fasterxml/jackson/databind/failing/RecordWithJsonNaming3102Test.java

```diff
@@ -1,15 +1,16 @@
 package com.fasterxml.jackson.databind.failing;
 
+import org.junit.Ignore;
 import com.fasterxml.jackson.annotation.JsonCreator;
-
 import com.fasterxml.jackson.databind.BaseMapTest;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.ObjectReader;
 import com.fasterxml.jackson.databind.PropertyNamingStrategies;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
 
-// [databind#3102]: fails on JDK 16 which finally blocks mutation
-// of Record fields.
+//[databind#3102]: fails on JDK 16 which finally blocks mutation
+//of Record fields.
+@Ignore
```

Comment:
you don't need to ignore tests that are in the failing package - the pom.xml skips tests in this package